streams/src/test/java/org/apache/kafka/streams/smoketest/SmokeTestUtil.java

```diff
@@ -37,14 +37,14 @@
     public final static long START_TIME = 60000L * 60 * 24 * 365 * 30;
     public final static int END = Integer.MAX_VALUE;
 
-    public static <T> ProcessorSupplier<String, T> printProcessorSupplier(final String topic) {
+    public static ProcessorSupplier<Object, Object> printProcessorSupplier(final String topic) {
```

Comment:
I feel this change as well as the ones in `SmokeTestClient.java` is not necessary?